Sources/Kafka/KafkaError.swift

```diff
@@ -116,7 +117,8 @@ public struct KafkaError: Error, CustomStringConvertible, @unchecked Sendable {
         )
     }
 
-    static func topicCreation(
+    @_spi(Internal)
+    public static func topicCreation(
```

Comment:
Since this error can only occur you can move it to an `extension KafkaError` in the `KafkaTestUtils` target and remove the `@spi` annotation